api/signup_test.go

```diff
@@ -150,6 +156,8 @@ func (ts *SignupTestSuite) TestWebhookTriggered() {
 	ts.API.handler.ServeHTTP(w, req)
 	assert.Equal(http.StatusOK, w.Code)
 	assert.Equal(1, callCount)
+
+	privateIPBlocks = append([]*net.IPNet{localhost}, privateIPBlocks...)
```

Comment:
is that needed since there is an unshift above?

Reply:
Nope! Good catch, forgot to remove it.